src/Dev/BuildTask.php

```diff
@@ -62,9 +70,17 @@ abstract public function run($request);
     /**
      * @return bool
      */
-    public function isEnabled()
+    public function isEnabled(): bool
     {
-        return $this->enabled;
+        $isEnabled = $this->config()->get('enabled');
+        $isDefaultEnabled = Config::inst()->get(__CLASS__, 'default_enabled');
```

Comment:
As mentioned above, we probably do not want this

Reply:
See comment above.